src/main/kotlin/link/infra/packwiz/installer/ui/gui/ExceptionListWindow.kt

```diff
@@ -112,6 +124,14 @@ class ExceptionListWindow(eList: List<ExceptionDetails>, future: CompletableFutu
 							this@ExceptionListWindow.dispose()
 						}
 					})
+					add(JButton("Open missing mods").apply {
```

Comment:
Could this be made to only show up if there are missing mods? It'd be confusing if you get some unrelated errors and a button that does nothing appears.